docs/howto/add_root_key.md

```diff
@@ -0,0 +1,56 @@
+# How to use add root key
+
+Add the root key to your API is quite simple with AMS. The **Adapter** is what determines the format of your JSON response. The default adapter is the ```FlattenJSON``` which doesn't have the root key, so your response is something similar to:
+
+```json
+{
+  id: 1,
+  title: "Awesome Post Tile",
+  content: "Post content"
+}
+```
+
+In order to add the correspondent root key you need to use the ```JSON``` Adapter, you can change this in an initializer:
+
+```ruby
+ActiveModel::Serializer.config.adapter = :json_api
+```
+
+or
+
+```ruby
+ActiveModel::Serializer.config.adapter = ActiveModel::Serializer::Adapter::Json
```

Comment:
Which makes me think that it'd make sense to create a doc on "How to create your own adapter".